### ISSUE TITLE
Players should be able to break the thrown bones.

### ISSUE THREAD

**Issue by Nodding** [fornclake/TetraForce#245]
**Description**

A sword swipe should cut through a thrown bone and destroy the projectile.

**Comment by josephbmanley** [comment]
I am a fan of this idea. Though, in that case, it might be worthwhile to add more enemies to increase difficulty.

**Comment by josephbmanley** [comment]
@Elginive what's your thought on this? I don't think it would be that difficult of a change.

**Comment by Elginive** [comment]
I don't think it would be too hard to Implement @Nodding might actually be able to handle it- it would only really need the cut() function, not unlike the pots or grass- very easy.

**Comment by josephbmanley** [comment]
Okay, in that case @Nodding, it is up to you! :rofl:

**Comment by Nodding** [comment]
I shall live out my dream
